OpenXmlPowerToolsExamples/TextReplacer01/TextReplacer01.cs

```diff
@@ -1,39 +1,36 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text;
 using DocumentFormat.OpenXml.Packaging;
 using OpenXmlPowerTools;
+using System;
+using System.IO;
 
-class TestPmlTextReplacer
+internal class TestPmlTextReplacer
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)